File: ExerciciosJavaScript(LógicadeProgramacao)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio12.js
/* 12) Escreva um algoritmo para ler uma temperatura em graus Fahrenheit, calcular e escrever o valor
correspondente em graus Celsius (baseado na fórmula abaixo):
    C           F - 32
---------- = -----------
    5             9    */


var grausF = parseInt(prompt("Digite o valor em graus Fahrenheit:"))
var grausC = (grausF - 32) / 1.8
alert("O valor convertido em Graus Celsius é: " + grausC)

File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio12.java
package exerciciosFaccat;

import java.util.Scanner;

/* 12) Escreva um algoritmo para ler uma temperatura em graus Fahrenheit, calcular e escrever o valor
correspondente em graus Celsius*/

public class Exercicio12 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("Digite o valor de Graus Fahrenheit para converter em graus Celsius:");
		Float grausF = sc.nextFloat();

		Float grausC = (grausF - 32) / 9;
		grausC = grausC * 5;

		System.out.println("O valor de " + grausF + " graus Fahrenheit convertido para Celsius é " + grausC + " graus Celsius");
		
		sc.close();
	}
}
